frontend/src/screens/signup.tsx

```diff
@@ -46,8 +46,8 @@ export default function SignUp(): JSX.Element{
     if (checkInput()) {
       try {
         await firebaseService.createUser(email, password);
-        await createUser({
-          _id: new ObjectID,
+        await UsersAPI.createUser({
+          _id: (new ObjectID).toHexString(),
           full_name: fullName,
           email,
           badges: [],
```

Comment:
I feel like the frontend shouldn't really have to initialize badges and other objects as empty, should be handled in the backend, the frontend's single responsibility should be to provide full name, email and password, while the backend initializes the other objects for the user. This is only a suggestion though!